text/XXXX-browser-traces.md

```diff
@@ -0,0 +1,98 @@
+* Start Date: 2022-09-28
+* RFC Type: decision
+* RFC PR: <link>
+
+# Summary
+
+This RFC proposes to require a frontend SDK to retain a trace that it continues
+until the SDK naturally ends the user session.  This change is primarily to the
+browser trace model to better support dynamic sampling and to create a more
+coherent user experience.
```

Comment:
I believe this affects mobile as well since the concept is similar to the browser trace model, right?

Reply:
Yes it will also affect mobile - by making ui.load and resulting interaction and navigation transactions linked.